### PR TITLE
[docs] Make Quickstart instructions more clear

### PR DESCRIPTION
**Pull request by jackkoenig**
### Contributor Checklist

- [ ] Did you add Scaladoc to every public function/method?
- [ ] Did you add at least one test demonstrating the PR?
- [x] Did you delete any extraneous printlns/debugging code?
- [x] Did you specify the type of improvement?
- [x] Did you add appropriate documentation in `docs/src`?
- [x] Did you request a desired merge strategy?
- [ ] Did you add text to be included in the Release Notes for this change?

#### Type of Improvement

- Documentation or website-related

#### Desired Merge Strategy

- Squash

### Reviewer Checklist (only modified by reviewer)
- [ ] Did you add the appropriate labels? (Select the most appropriate one based on the "Type of Improvement")
- [ ] Did you mark the proper milestone (Bug fix: `3.6.x`, `5.x`, or `6.x` depending on impact, API modification or big change: `7.0`)?
- [ ] Did you review?
- [ ] Did you check whether all relevant Contributor checkboxes have been checked?
- [ ] Did you do one of the following when ready to merge:
  - [ ] Squash: You/ the contributor `Enable auto-merge (squash)`, clean up the commit message, and label with `Please Merge`.
  - [ ] Merge: Ensure that contributor has cleaned up their commit history, then merge with `Create a merge commit`.

### CODE REVIEW

**Review by mwachs5 — APPROVED** [review]
clearer for copy-pasters like me :)

File: docs/src/installation.md [hunk 1]
```diff
@@ -9,10 +9,19 @@ Chisel is a [Scala](https://www.scala-lang.org/) library and compiler plugin.
 
 ## Quickstart with Scala CLI
 
-The easiest way to install Chisel is to [install Scala CLI](https://scala-cli.virtuslab.org/install) to build and run the Chisel template:
+The easiest way to use Chisel is to first [install Scala CLI](https://scala-cli.virtuslab.org/install).
```
Comment (mwachs5) [inline docs/src/installation.md:12]:
```suggestion
The easiest way to use Chisel is to:

 1.  First [install Scala CLI](https://scala-cli.virtuslab.org/install).
```

File: docs/src/installation.md [hunk 2]
```diff
@@ -9,10 +9,19 @@ Chisel is a [Scala](https://www.scala-lang.org/) library and compiler plugin.
...
 
+Then, download the Chisel template.
```
Comment (mwachs5) [inline docs/src/installation.md:14]:
```suggestion
2. Then, download the Chisel template.
```

File: docs/src/installation.md [hunk 3]
```diff
@@ -9,10 +9,19 @@ Chisel is a [Scala](https://www.scala-lang.org/) library and compiler plugin.
...
+
+This is easiest on the command-line (works on Linux, MacOS, and Windows Subsystem for Linux (WSL)):
 ```bash
 curl -O -L https://github.com/chipsalliance/chisel/releases/latest/download/chisel-template.scala
+```
+
+Or alternatively you can download the template directly from [this link](https://github.com/chipsalliance/chisel/releases/latest/download/chisel-template.scala).
+
+Finally, use Scala CLI to compile and run the template:
```
Comment (mwachs5) [inline docs/src/installation.md:23]:
```suggestion
3. Finally, use Scala CLI to compile and run the template:
```

**Review by mwachs5 — COMMENTED** [review]
suggested some numberings to make the steps more divide, but either way this PR is more clear

File: docs/src/installation.md [hunk 4]
```diff
@@ -9,10 +9,21 @@ Chisel is a [Scala](https://www.scala-lang.org/) library and compiler plugin.
 
 ## Quickstart with Scala CLI
 
-The easiest way to install Chisel is to [install Scala CLI](https://scala-cli.virtuslab.org/install) to build and run the Chisel template:
+The easiest way to use Chisel is to:
 
+ 1. First, [install Scala CLI](https://scala-cli.virtuslab.org/install).
```
Comment (jackkoenig) [inline docs/src/installation.md:14]:
```suggestion
1. First, [install Scala CLI](https://scala-cli.virtuslab.org/install).
```

File: docs/src/installation.md [hunk 5]
```diff
@@ -9,10 +9,21 @@ Chisel is a [Scala](https://www.scala-lang.org/) library and compiler plugin.
...
+1. First, [install Scala CLI](https://scala-cli.virtuslab.org/install).
+
+2. Then, download the Chisel template.
+
+This is easiest on the command-line (works on Linux, MacOS, and Windows Subsystem for Linux (WSL)):
 ```bash
 curl -O -L https://github.com/chipsalliance/chisel/releases/latest/download/chisel-template.scala
+```
+
+Or alternatively you can download the template directly from [this link](https://github.com/chipsalliance/chisel/releases/latest/download/chisel-template.scala).
```
Comment (jackkoenig) [inline docs/src/installation.md:23]:
```suggestion
Alternatively you can download the template directly from [this link](https://github.com/chipsalliance/chisel/releases/latest/download/chisel-template.scala).
```

**Pull request merged**